lib/modules/manager/helmv3/artifacts.spec.ts

```diff
@@ -111,6 +112,45 @@ describe('modules/manager/helmv3/artifacts', () => {
     expect(execSnapshots).toMatchSnapshot();
   });
 
+  it('returns null if only "generated" is changed', async () => {
+    fs.readLocalFile.mockResolvedValueOnce(stripIndent`
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce(codeBlock`
```

we prefer `codeBlock`